src/openms/source/ANALYSIS/RNPXL/RNPxlReport.cpp

```diff
@@ -270,7 +270,7 @@ namespace OpenMS
             }
             else // place it anywhere
             {
-              for (auto a : aa)
+              for (auto a : aa) // NOTE: performs copy, this cannot possibly work!
```

Comment:
@timosachsenberg I dont think this can possibly work, can you confirm?  I think this should be a non-const ref, right?

```suggestion
              for (auto& a : aa) // NOTE: performs copy, this cannot possibly work!
```